apps/extension/client/snippets/xstate.code-snippets

```diff
@@ -0,0 +1,20 @@
+{
+  "XState Machine": {
+    "scope": "javascript,javascriptreact,vue",
+    "prefix": "xsm",
+    "body": [
+      "import { createMachine } from 'xstate';",
+      "const ${1:nameOf}Machine = createMachine({\n\tid: '${1:nameOf}',\n\tinitial: '${2:initialState}',\n\tstates: {\n\t\t${2:initialState}: {$0},\n\t}\n});"
+    ],
+    "description": "Outline for XState Machine"
+  },
+  "XState Typegen Machine": {
+    "scope": "typescript,typescriptreact,vue",
+    "prefix": "xsm",
```

Comment:
@mellson both of those snippets use the same prefix and are activated in the `vue` scope. Does it create a conflict? 🤔

Reply:
Good question! No it doesn't conflict, both options are shown and the user can pick one based of the name.